packages/tailwindcss/src/design-system.ts

```diff
@@ -13,6 +13,8 @@ export type DesignSystem = {
   utilities: Utilities
   variants: Variants
 
+  important: string | boolean
```

Comment:
Is this overloading of the `important` option a v3-ism that creeps in? Wondering if this should be two separate options instead.

Is there a world where you'd maybe want both (I guess the answer is no because we would rather have people not use these at all?).

Reply:
Yeah it is kinda a v3-ism. I'd be cool with them being separate options. Originally I wanted selector to operate like `#app { @tailwind utilities; }` but after talking with @RobinMalfait we figured it's probably useful for intellisense to actually see that the important selector is in use — like how it works in v3.

But yeah in an ideal world people don't use these at all unless they absolutely have to.